src/app/loadout-builder/processWorker/process.ts

```diff
@@ -346,51 +346,6 @@ export function process(
   return { sets: flattenSets(finalSets), combos, combosWithoutCaps, statRanges };
 }
 
-const emptyStats: number[] = new Array(_.size(statHashes)).fill(0);
-
-/**
- * Generate all possible stat mixes this item can contribute from different perk options,
- * expressed as comma-separated strings in the same order as statHashes.
- */
-function getStatMix(item: ProcessItem, assumeMasterwork: boolean, orderedStatValues: number[]) {
```

Comment:
Used for finding Y1 armour stats, now just using the getStatValuesWithMWProcess function directly.